src/hooks/index.ts

```diff
@@ -1005,3 +974,79 @@ export const useBalances = (wallet: YoroiWallet): YoroiAmounts => {
 
   return Utxos.toAmounts(utxos, primaryTokenId)
 }
+
+export const useRefetchOnFocus = (refetch: () => unknown, canRefetch = true) => {
```

Comment:
can this be replaced with `useFocusEffect`?

Reply:
what is `canRefetch` for? its never used anywhere

Reply:
`useFocusEffect` has to take a memoized callback
this implementation will call the callback on ever render